package.json

```diff
@@ -37,7 +37,7 @@
     "eslint-plugin-jest": "^27.4.2",
     "eslint-plugin-jest-dom": "^5.1.0",
     "eslint-plugin-jest-playwright": "^0.9.0",
-    "eslint-plugin-prettier": "^3.3.1",
+    "eslint-plugin-prettier": "^5.0.0",
```

Comment:
Upgraded eslint-plugin-prettier to v5 to resolve error based on [this issue](https://github.com/prettier/eslint-plugin-prettier/issues/562)

<img width="1564" alt="Screenshot 2024-03-01 at 12 36 36 PM" src="https://github.com/binary-com/eslint-config-deriv/assets/103016120/35fb3e44-8e26-498a-ad96-730813e0a4f6">